server.go

```diff
@@ -40,6 +44,7 @@ func (s *server) Init(opt ...grpc.ServerOption) {
 	s.initAuthService()
 	s.initRepositories()
 	s.initAccountsService()
+	s.initGroupService()
```

Comment:
Nitpicking but you refer to `groupService` when it should be `groupsService` just like its `accountService` and not `accountService`.